src/types/nvi.types.ts

```diff
@@ -98,3 +99,7 @@ export interface NviPeriod {
 export interface NviPeriodResponse {
   periods: NviPeriod[];
 }
+
+export interface CandidateOffsetState {
+  offsetNextCandidate?: number;
```

Comment:
lurer på om dette ble feil.. når routen først har en state så har den vel alltid `offsetNextCandidate` (?)
```suggestion
  offsetNextCandidate: number;
```